agents/ocf/HealthIOWait.in

```diff
@@ -175,13 +184,17 @@ if [ -z "${OCF_RESKEY_yellow_limit}" ] ; then
   OCF_RESKEY_yellow_limit=10
 fi
 
+if [ -z "${OCF_RESKEY_dampening}" ]; then
+  OCF_RESKEY_dampening="5s"
+fi
```

Comment:
Not a problem, but FYI, this could also be done like

    : ${OCF_RESKEY_CRM_dampening:=5s}

Reply:
If ${OCF_RESKEY_CRM_dampening:=5s} is used to replace the preceding lines of code, the attrd_updater command will still use 5s when the dampening is changed

Reply:
What is the difference between OCF_RESKEY_CRM_dampening and OCF_RESKEY_dampening? Do you have any information on this issue? Thanks

Reply:
> If ${OCF_RESKEY_CRM_dampening:=5s} is used to replace the preceding lines of code, the attrd_updater command will still use 5s when the dampening is changed

The behavior should be identical -- the `:=` syntax sets the value only if empty.

> What is the difference between OCF_RESKEY_CRM_dampening and OCF_RESKEY_dampening? Do you have any information on this issue? Thanks

`OCF_RESKEY_dampening` is the resource agent's dampening parameter. `OCF_RESKEY_CRM_dampening` would just be a regular variable (Pacemaker wouldn't set or use it); `OCF_RESKEY_CRM_meta_dampening` would be the dampening resource meta-attribute, which doesn't exist, so it would just be a regular variable too.

Reply:
Whoops, I just noticed I was the one that added "CRM" in there unnecessarily :) That was an accident, I meant `OCF_RESKEY_dampening`

Reply:
"OCF_RESKEY_dampening"  is a simpler and more standardized way to write it.